src/test/java/io/confluent/connect/jdbc/dialect/BaseDialectTest.java

```diff
@@ -413,7 +413,12 @@ public void bindFieldPrimitiveValues() throws SQLException {
     verifyBindField(++index, Schema.FLOAT64_SCHEMA, 42d).setDouble(index, 42d);
     verifyBindField(++index, Schema.BYTES_SCHEMA, new byte[]{42}).setBytes(index, new byte[]{42});
     verifyBindField(++index, Schema.BYTES_SCHEMA, ByteBuffer.wrap(new byte[]{42})).setBytes(index, new byte[]{42});
-    verifyBindField(++index, Schema.STRING_SCHEMA, "yep").setString(index, "yep");
+    if (dialect instanceof OracleDatabaseDialect) {
+      verifyBindField(++index, Schema.STRING_SCHEMA, "yep").setNString(index, "yep");
```

Comment:
I think this can be pushed down into the dialect-specific test class by pulling this out into a separate test method, and overriding it in the oracle class.